### ISSUE TITLE
[3.x] Embed conditional fields in components

### ISSUE THREAD

**Issue by haringsrob** [area17/twill#1630]
## Summary

With the new blade components we can do something like this:

```php
<x-twill:x-twill::checkbox
    name="checkbox"
    label="checkbox"
/>

<x-twill::input
    name="test"
    label="Test"
    :show-if="['checkbox', true]"
/>

<x-twill::input
    name="test2"
    label="Test2"
    :hide-if="['checkbox', false]"
/>
```

**Comment by haringsrob** [comment]
This will work if we also implement it into the form-builder components.

**Comment by ifox** [comment]
This has been released in Twill 3.1.0 with #2323.
